<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="top_Area">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="modelAdder">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="dataEdit">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="xLabel">
                <property name="text">
                 <string>X:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="xLineEdit"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="yLabel">
                <property name="text">
                 <string>Y:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="yLineEdit"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="zLabel">
                <property name="text">
                 <string>Z:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="zLineEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="yawLabel">
                <property name="text">
                 <string>Yaw:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="yawLineEdit"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="rollLabel">
                <property name="text">
                 <string>Roll:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="rollLineEdit"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="pitchLabel">
                <property name="text">
                 <string>Pitch:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="pitchLineEdit"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="timeSecondsLabel">
                <property name="text">
                 <string>Time(Seconds)</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="timeSecondsLineEdit"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="modelButtons">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::MinimumExpanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="insertNextBtn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Insert Next</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushToEndBtn">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Push To End</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QOpenGLWidget" name="openGLVideo">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>300</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="bottom_Area">
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item>
         <widget class="QLabel" name="xCurrent">
          <property name="text">
           <string>X: {current.x}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="yCurrent">
          <property name="text">
           <string>y: {current.y}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="zCurrent">
          <property name="text">
           <string>z: {current.z}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="yawCurrent">
          <property name="text">
           <string>yaw: {current.yaw}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pitchCurrent">
          <property name="text">
           <string>pitch: {current.pitch}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="rollCurrent">
          <property name="text">
           <string>roll: {current.roll}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="timeCurrent">
          <property name="text">
           <string>time: time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deletePointBtn">
          <property name="text">
           <string>Delete Point</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>946</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
